doc/requirements-and-limitations.md

```diff
@@ -30,8 +30,7 @@ The `SUPER` privilege is required for `STOP SLAVE`, `START SLAVE` operations. Th
 
 - MySQL 5.7 `JSON` columns are not supported. They are likely to be supported shortly.
 
-- The two _before_ & _after_ tables must share some `UNIQUE KEY`. Such key would be used by `gh-ost` to iterate the table.
-  - As an example, if your table has a single `UNIQUE KEY` and no `PRIMARY KEY`, and you wish to replace it with a `PRIMARY KEY`, you will need two migrations: one to add the `PRIMARY KEY` (this migration will use the existing `UNIQUE KEY`), another to drop the now redundant `UNIQUE KEY` (this migration will use the `PRIMARY KEY`).
+- The two _before_ & _after_ tables must share some `UNIQUE KEY`. Such key would be used by `gh-ost` to iterate the table. See [Read more](shared-key.md)
 
 - The chosen migration key must not include columns with `NULL` values.
```

Comment:
Okay. I think this is really wordy, though, because it confused me. Since we're doing things in note form, maybe something like:

- The migration key must not include columns with `NULL` values. This means either:
  1. The columns are `NOT NULL`, or
  2. The columns are nullable but don't contain any NULL values. 

- `gh-ost` will try to pick a migration key with non-nullable columns. It will not run if the only `UNIQUE KEY` includes nullable columns. 
  - You may override this via `--allow-nullable-unique-key` but **make sure** there are no actual `NULL` values in those columns. Existing `NULL` values can't guarantee data integrity on the migrated table.